content/docs/2.7/scalers/azure-data-explorer.md

```diff
@@ -40,6 +44,7 @@ triggers:
 - `tenantId` - Id of the Azure AD tenant.
 - `clientId` - Id of the Azure AD application.
 - `clientSecret` - Password of the Azure AD application.
+- `activeDirectoryEndpoint` - Active Directory endpoint of the cloud environment. (Required when `cloud` is set to `Private`, e.g. `https://login.chinacloudapi.cn/` for `AzureChinaCloud`).
```

Comment:
Can you document the cloud configuration option here please?

Reply:
Whoops, bad oversight by me.

Reply:
Done.

Reply:
That's what reviews are for, thanks!